### DISCUSSION TITLE
For fun: Geekbench 3 on Arch

### DISCUSSION THREAD

**Discussion by emmatyping** [copy/v86#542] [General]
Out of interest, I ran geekbench 3 on my Windows 11 desktop in Firefox 95.0a1 using the Arch Linux OS image. I have an AMD R9 3900X.

The results are here: https://browser.geekbench.com/v3/cpu/8967298

The overall performance is roughly equivalent to a Intel Pentium T4400.

Just thought this would be an interesting reference point, thanks for the cool project :)

**Comment by copy** [comment]
Nice find. The performance looks pretty good I'd say. I'll probably include a script that downloads the benchmark and runs it when I update the Arch image the next time.

**Comment by spetterman66** [comment]
https://browser.geekbench.com/v3/cpu/9087413 slightly beaten

**Comment by ercan1104** [comment]
I made benchmark on Geekbench 2:
Arch Linux: https://browser.geekbench.com/geekbench2/2736093
Tiny Core Linux: https://browser.geekbench.com/geekbench2/2730912
